src/Security/MemberAuthenticator/LostPasswordHandler.php

```diff
@@ -227,4 +249,21 @@ protected function sendEmail($member, $token)
             ->setTo($member->Email);
         return $email->send();
     }
+
+    /**
+     * Avoid information disclosure by displaying the same status, regardless wether the email address actually exists
+     *
+     * @param array $data
+     * @return HTTPResponse
+     */
+    protected function redirectToSuccess(array $data)
+    {
+        $link = Controller::join_links(
+            $this->link('passwordsent'),
```

Comment:
Should passwordsent be a configuration?

Reply:
The difficulty is that this class is a mixture of configuration and concrete implementation. By putting it into its own method we can overload it easily in children, which is what I opted for in the end

Reply:
Hmmm, maybe both would be a better option? @tractorcow what do you think?

Reply:
Seems ok as it is. It's hard-coded in the same class that implement that exact url handler, so it's not coupled incorrectly.

Perhaps if linking to Security/passwordsent then maybe. :)